From ec4972802c254718065be687c88149a38e61e38e Mon Sep 17 00:00:00 2001
From: Ethan Gafford <egafford@redhat.com>
Date: Mon, 23 Mar 2015 15:26:51 -0400
Subject: [PATCH] Include launch_command.py in MANIFEST.in

Stopgap to repair Spark until next upstream Juno release.
Resolves: rhbz#153352
---
 MANIFEST.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/MANIFEST.in b/MANIFEST.in
index bd7117f..8dfc9bb 100644
--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -36,6 +36,7 @@ include sahara/plugins/spark/resources/*.sh
 include sahara/plugins/spark/resources/*.template
 include sahara/resources/*.heat
 include sahara/service/edp/resources/*.xml
+include sahara/service/edp/resources/launch_command.py
 include sahara/swift/resources/*.xml
 include sahara/tests/unit/plugins/vanilla/hadoop2/resources/*.txt
 include sahara/tests/unit/resources/*.heat
